pkg/csi/service/vanilla/controller.go

```diff
@@ -760,6 +757,22 @@ func (c *controller) ControllerExpandVolume(ctx context.Context, req *csi.Contro
 	log := logger.GetLogger(ctx)
 	log.Infof("ControllerExpandVolume: called with args %+v", *req)
 
+	var volumePath string
```

Comment:
I think we can avoid creating this variable.

Reply:
yes removed it.